packages/grafana-data/src/types/templateVars.ts

```diff
@@ -66,11 +65,10 @@ export interface AdHocVariableModel extends BaseVariableModel {
   baseFilters?: AdHocVariableFilter[];
 }
 
-export interface GroupByVariableModel extends BaseVariableModel {
+export interface GroupByVariableModel extends VariableWithOptions {
```

Comment:
@ashharrison90 / @torkelo  - did this change, think we can skip the `defaultOptions` in the variable model, and just rely on `options` that we already have in the base type. This is a persistence schema that's mapped to the `defaultOptions` in the GroupByVariable scene object when it's created.

Reply:
no worries 😄 a lot of the time we don't know something until we start working on it